index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
             </ul>
         </div>
       </section>
+      <section class="works">
+        <div class="recent-works">
+          <h2 id="worksTitle">My Recent Works</h2>  
+        </div>
+        <div id="grid-container">
+            <div class="recent-project">
+              <img class="project-img" src="img/Img-Placeholder.png" alt="img"/>
+              <div class="right-block">
+                 <h2 class="title-post">Multi-Post Stories</h2>
+                 <p class="posts-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                 <ul class="tech-skills">
+                  <li class="skillset">CSS</li>
+                  <li class="skillset">HTML</li>
+                  <li class="skillset">Bootstrap</li>
+                  <li class="skillset">Ruby</li>
+                 </ul>
+                 <button class="project-view"><p>See project</p></button>
+                </div>
+            </div>
+               <!--flexbox individual cards-->
+            <div class="cards-box">
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
```

Comment:
- [ ] According to the project requirement you should stick to the design like color, background, spacing, etc.
- [ ] The image should be in the center of the page please fix it.
- [ ] Please increase the font weight of the heading and also use the same color for the `p` tag
- [ ] Also use the same color for the list that is `color: #091E42;`. I notice that you have different colors for buttons please update is according to the Figma design and also remove the border to match the design. Please use the same background color also.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236485685-0586c973-d59b-4128-b59d-a7116d883f7c.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236485718-be84e1b6-2db1-4842-9a93-04834cb17529.png)